documentation/project-proposal.md

```diff
@@ -131,49 +131,37 @@ Given I am a logged-in user,\
 When I select to remove an existing budget,\
 Then the budget should be successfully removed and not show on the budget tab.
 
-4. As a user, I want to set budgets for specific categories like food and travel so that I can allocate funds to different areas of my life.\
-**Acceptance Criteria:**\
-Given I am a logged-in user,\
-When I set a budget for a specific category,\
-Then I should see the correct budget set for each category.
-
-5. As a user, I want to see how much of my budget I have used so that I can adjust my spending habits.\
+4. As a user, I want to see how much of my budget I have used so that I can adjust my spending habits.\
 **Acceptance Criteria:**\
 Given I am a logged-in user,\
 When I navigate to the budget tab,\
 Then I should see already existing budget and their progress.
 
 ### Expense Analytics
 
-1. As a user, I want visual insights into my expenses so that I can quickly understand my spending habits.\
+1. As a user, I want visual insights into my expenses from the first day of the month to the current day of the month so that I can quickly understand my spending habits.\
 **Acceptance Criteria:**\
 Given I am a logged-in user,\
 When I access the app's analytics dashboard,\
 Then I should be able to see spending trends and patterns displayed through graphs and charts.
 
-2. As a user, I want to analyze my expenses for specific time frames so that I can review my spending over set periods.\
-**Acceptance Criteria:**\
-Given I am a logged-in user,\
-When I set a time interval on the analytics board,\
-Then the analytics should correctly update and show the appropriate data.
-
-3. As a user, I want to compare expenses over different time frames so that I can identify trends or changes in my spending.\
+2. As a user, I want to compare expenses over two separate past months so that I can identify trends or changes in my spending.\
 **Acceptance Criteria:**\
 Given I am a logged-in user,\
 When I choose the "Compare" option in the analytics dashboard,\
-Then I should be able to see spending trends and habits of different time frames side by side.
+Then I should be able to see spending trends and habits of two past months side by side.
 
-4. As a user, I want to see spending patterns for each category so that I can gain insights into where most of my money goes.\
+3. As a user, I want to see spending patterns for the current month each category so that I can gain insights into where most of my money goes.\
```

Comment:
```suggestion
3. As a user, I want to see spending patterns of each category for the current month so that I can gain insights into where most of my money goes.\
```

Reply:
of each category in the current month sounds better?